Gemfile.lock

```diff
@@ -281,4 +281,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   1.14.5
+   1.14.6
```

Comment:
None of our rubies are pinned to bundler 1.14.6 yet, so this shouldn't be changed here.